build.sbt

```diff
@@ -571,3 +570,12 @@ lazy val update2_12 = Seq(
     }
   }
 )
+
+lazy val xlint = Seq(
+  scalacOptions += {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 12)) => "-Xlint:-unused,_"
```

Comment:
This now warns on unused parameters, vals and implicits according to `scalac -xlint:help`